java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -290,6 +293,58 @@ public T decrBy(@NonNull String key, long amount) {
         return getThis();
     }
 
+    /**
+     * Retrieve the value associated with <code>field</code> in the hash stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/hget/">redis.io</a> for details.
+     * @param key The key of the hash.
+     * @param field The field in the hash stored at <code>key</code> to retrieve from the database.
+     * @return Command Response - The value associated with <code>field</code>, or <code>null</code>
+     *     when <code>field
+     *     </code> is not present in the hash or <code>key</code> does not exist.
+     */
+    public T hget(@NonNull String key, @NonNull String field) {
+        ArgsArray commandArgs = buildArgs(key, field);
+
+        protobufTransaction.addCommands(buildCommand(HashGet, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Sets the specified fields to their respective values in the hash stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/hset/">redis.io</a> for details.
+     * @param key The key of the hash.
+     * @param fieldValueMap A field-value map consisting of fields and their corresponding values to
+     *     be set in the hash stored at the specified key.
+     * @return Command Response - The number of fields that were added.
+     */
+    public T hset(@NonNull String key, @NonNull Map<String, String> fieldValueMap) {
+        ArgsArray commandArgs =
+                buildArgs(ArrayUtils.addFirst(convertMapToArgArray(fieldValueMap), key));
+
+        protobufTransaction.addCommands(buildCommand(HashSet, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Removes the specified fields from the hash stored at <code>key</code>. Specified fields that do
```

Comment:
```suggestion
     * Remove the specified fields from the hash stored at <code>key</code>. Specified fields that do
```

Reply:
@Yury-Fridlyand we should stick with whatever tense they have in Python/Node for consistency sake.